services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -850,6 +854,45 @@ instance SetFeatureConfig db MLSConfig where
   setConfigForTeam tid wsnl = do
     persistAndPushEvent @db tid wsnl
 
+instance GetFeatureConfig db ExposeInvitationURLsToTeamAdminConfig where
+  getConfigForServer =
+    pure $
```

Comment:
```suggestion
    -- we could look at the galley settings, but we don't have a team here, so there is not much else we can say.
    pure $
```

(only if you think this may clarify things for other people than present me.)